lib/webauthn/authenticator_response.rb

```diff
@@ -24,7 +24,7 @@ def initialize(client_data_json:)
       @client_data_json = client_data_json
     end
 
-    def verify(expected_challenge, expected_origin = nil, user_verification: nil, rp_id: nil)
+    def verify(expected_challenge, expected_origin = nil, user_verification: nil, rp_id: nil, fido_app_id: nil)
```

Comment:
I believe determining RP ID or AppID is RP implementation's job, by something like:

```
response.verify(
  ...,
  rp_id: response.u2f_appid? ? rp_id : fido_app_id
)
```

I've done U2F → WebAuthn migration at https://github.com/sorah/clarion/pull/1 with the current implementation in past, without modifying webauthn-ruby.

Reply:
Great suggestion and thank you for that reference PR, that's helpful!